templates/mapTpl.go

```diff
@@ -0,0 +1,42 @@
+package templates
+
+import (
+	"github.com/bigbluedisco/protoc-gen-zap/zap"
+	pgs "github.com/lyft/protoc-gen-star"
+)
+
+type mapData struct {
+	MapName   string
+	Getter    string
+	AddFunc   string
+	FormatKey string
+	IsStars   bool
+}
+
+func newMapData(f pgs.Field, obsType zap.ObfuscationType) mapData {
+	formatKey := "key"
+	keyType := f.Type().Key().ProtoType()
+	if isSimple(keyType) && keyType != pgs.StringT {
+		formatKey = "strconv.Itoa(int(key))"
+	}
+
+	return mapData{
+		MapName:   name(f),
+		Getter:    getter(f.Name(), f.Type()),
+		FormatKey: formatKey,
+		IsStars:   obsType == zap.ObfuscationType_STARS,
+		AddFunc:   simpleAddFunc(f.Type().Element()),
+	}
+}
+
+const mapTpl = `
+{{ if .IsStars }}
+for key := range {{ .Getter }} {
+		o.AddString("{{ .MapName }}_" + key, "***")
```

Comment:
I think this is missing "FormatKey" no ?

Reply:
it works, here is the result of the test

```
	for key := range c.GetMyHiddenMap() {
		o.AddString("my_hidden_map_"+key, "***")
	}

	for key, value := range c.GetMyMap() {
		o.AddUint32("my_map_"+key, value)
	}

```